sonar-html-plugin/src/main/java/org/sonar/plugins/html/lex/NormalElementTokenizer.java

```diff
@@ -33,9 +33,6 @@ public boolean consume(CodeReader codeReader, List<Node> nodeList) {
       return false;
     }
     int i = 1;
-    while (Character.isWhitespace(codeReader.charAt(i))) {
-      i++;
-    }
```

Comment:
This code is useless because it is incorrect in HTML to have something like ```<     html>``` compared to ```<html>```, right?